about.html

```diff
@@ -0,0 +1,126 @@
+                <li><a href="#">ticket</a></li>
+                <li><a href="#">shows</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="nav_link_ticket" href="#"> Get ticket</a></li>
+            </ul>
+            <ul class="nav_link">
+                <li><a class="nav_link_line" href="#">About</a></li>
+                <li><a class="nav_link_line" href="#">concert</a></li>
+                <li><a class="nav_link_line" href="#">ticket</a></li>
+            </ul>
+        </nav>
+
+    </header>
+    <main>
+        <section class="homepage">
+            <div class="homepage_content">
+                <div class="header_content">
+                    <h3>"welcome  y'all"</h3>
+                    <h1 class="h1_image">Monsters of rock world concert 2023</h1>
+                </div>
+                <div class="paragraph">
+                    <p>A great concert celebrating multiple talents from various counties <br>  
+                        which is taking place in gotham city on 15th october
+                        please y'all welcome to the greatest show of all time 
+                    </p>
+                </div>
+                <div class="paragraph_two">
+                    <p>Please contact us per Email of any further questions below</p>
+                    <h4>RecordLabel2023@submissions.org</h4>
+                </div>
+            </div>
+        </section>
+        <section class="work_section">
+            <div class="work_section_content">
+                <div class="main_program">
+                    <h2> create Record Label submissions 2023 logo</h2>
+                    <div class="line_program"></div>
+                    <p> 
+                        The logo Record Label submissions 2023 was decided through different season of concerts
+                    </p>
+                </div>
+                <div class="main_program_image">
+                    <img src="images/label.png" alt="">
+                </div>
+            </div>
+
+        </section>
+        <section class="features">
+            <div class="features_container">
+                <div class="featured_artists">
+                    <h2>See The Past Concerts</h2>
+                    <div class="line_program"></div>
+                    <p>
+                        Take a look at The Two previuos concerts that took place in berlin and calfornia
+                    </p>
+                </div>
+                <div class="past_concerts">
+                    <div class="first_past_concert">
+                        <h2>2018</h2>
+                        <p>Monsters of Rock 2018 in berlin</p>
+                    </div>
+                    <div class="second_past_concert">
+                        <h2>2018</h2>
+                        <p>Monsters of Rock 2018 in berlin</p>
+                    </div>
+                </div>
+                                    
```

Comment:
- [ ] On the mobile version the image is overlapping with the next section. Please adjust it accordingly. 🧐️


| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/217844322-d1ff0bee-5faf-4ba9-a792-f35f3f819911.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/217844338-0eeb0da0-3fc9-4602-89d1-050f689b4259.png" width="300">|